### PR TITLE
Load full roasts from CSV file

### PR DESCRIPTION
**Pull request by dmartinezgamboa**
- Add CVS file to load data through main.py
- Refactor and reorganize roast.py
- Refactors for variable names throughout app
- Update tests module to reflect changes

### CODE REVIEW

**Comment by bryanterichardson** [comment]
Whyd you quote the pr description?????? Do bullet points or something

File: roastme/data/roasts_full_file.csv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{user} go ahead and leave the server.
+{user} is trash.
+{user} nice jacket loser.
```
Comment (bryanterichardson) [inline roastme/data/roasts_full_file.csv:3]: lol

File: roastme/main.py [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
     f.close()
-    return data
+    return roast_pieces
 
 
 if __name__ == "__main__":
-    data = load_json_data(file_path='data.json')
-    roastme = RoastMeClient(data)
+    roasts_parts = load_json_data(file_path='data/roasts_parts_file.json')
+    roasts_full = load_csv_data(file_path='data/roasts_full_file.csv')
```
Comment (bryanterichardson) [inline roastme/main.py:23]: Okay I like this

File: roastme/tests/test_client.py [hunk 3]
```diff
@@ -14,23 +14,30 @@ def tearDown(self):
...
-            ],
-            "adjectives": [
-                "stupid.",
+        parts_data = MagicMock()
+        full_data = MagicMock()
+        roastme_instance = RoastMeClient(
+            roasts_parts=parts_data,
+            roasts_full=full_data)
+        self.assertEqual(roastme_instance.roasts_parts, parts_data)
+        self.assertEqual(roastme_instance.roasts_full, full_data)
```
Comment (bryanterichardson) [inline roastme/tests/test_client.py:23]: mmmm pretty good

File: roastme/tests/test_client.py [hunk 4]
```diff
@@ -14,23 +14,30 @@ def tearDown(self):
...
+
+    def test_finds_user(self):
```
Comment (bryanterichardson) [inline roastme/tests/test_client.py]: NIT: should be named something like `test_find_tagged_user_finds_user` where you prepend `test_`, include the full function name, then what youre testing

File: roastme/tests/test_client.py [hunk 5]
```diff
@@ -14,23 +14,30 @@ def tearDown(self):
...
+            "<@discord_user_3>"
             ]
-        }
-        user = '<@DiscordUser>'
-        roastme = RoastMeClient(data=data)
-        result = roastme.create_insult(user)
-        self.assertIsInstance(result, str)
+
+        result = roastme.find_tagged_user(message_author, message_mentions)
+        self.assertEqual(result, "<@discord_user_2>")
```
Comment (bryanterichardson) [inline roastme/tests/test_client.py:43]: nice

**Review by bryanterichardson — APPROVED** [review]
No blocking changes. Good job!!

File: roastme/data/roasts_full_file.csv [hunk 6]
```diff
@@ -0,0 +1,5 @@
...
+You literally smell {user}.
+Your head looks like a pineapple {user}.
```
Comment (bryanterichardson) [inline roastme/data/roasts_full_file.csv]: 🚫  no new line 👀 buuuut idk if csvs need that.

File: roastme/tests/test_client.py (hunk 4 above)
Comment (dmartinezgamboa) [reply roastme/tests/test_client.py]: Changed to a better name.

**Pull request merged**